pkg/trace/api/otlp.go

```diff
@@ -398,6 +398,54 @@ func marshalEvents(events ptrace.SpanEventSlice) string {
 	return str.String()
 }
 
+// marshalLinks marshals span links into JSON.
+func marshalLinks(links ptrace.SpanLinkSlice) string {
+	var str strings.Builder
+	str.WriteString("[")
+	for i := 0; i < links.Len(); i++ {
+		l := links.At(i)
+		if i > 0 {
+			str.WriteString(",")
+		}
+		t := l.TraceID()
+		str.WriteString(`{"trace_id":"`)
+		str.WriteString(hex.EncodeToString(t[:]))
```

Comment:
This makes sense and is probably what we want, but just making sure you are aware: this will be the full 128-bit ID, not the 64-bit ID conversion that we usually do. It will have to be converted temporarily in the backend in order to correlate the links.

Reply:
That's correct, we discussed this representation with the affected teams to make sure everyone was onboard with this approach.